geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py

```diff
@@ -692,7 +695,12 @@ def step11(self, step):
                 "docker-compose down; docker-compose up -d`.",
                 "- Test your application on 'http://localhost:8480/desktop'."
             ]
-
+        if self.options.demo:
+           message.append(
+               "Please run the following:"
+               "./docker-run rm /build/c2ctemplate-cache.json"
+               "./docker-run --home make --makefile=<user>.mk docker-compose.yaml"
+           )
```

Comment:
What's this hack?

Reply:
https://github.com/camptocamp/c2cgeoportal/wiki/New-release#in-demo